beets/util/artresizer.py

```diff
@@ -240,10 +259,15 @@ def convert_format(self, source, target, deinterlaced):
             return source
 
     @property
-    def can_compare(self):
+    def can_compare(self) -> bool:
         return self.version() > (6, 8, 7)
```

Comment:
Type checker flags this because it doesn't take into account that the version may be non-existent.
The original signature is a bit too complicated, see my previous comment.

Reply:
Mypy is not usable at the current stage as an indicator of typing correctness. See next comment.